plugins/reporters/trustsource/src/main/kotlin/TrustSourceModel.kt

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (C) 2021 The ORT Project Authors (see <https://github.com/oss-review-toolkit/ort/blob/main/NOTICE>)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.ossreviewtoolkit.plugins.reporters.trustsource
```

Comment:
Please document all classes and their properties in this file using [KDoc with inline markup](https://kotlinlang.org/docs/kotlin-doc.html#inline-markup). Esp. semantics and (in-)valid values should be documented (like, are strings allowed to be empty).

Reply:
Documentation is going to be added in a follow-up PR with the help of @gr-markin.